### PR TITLE
Update & Cleanup README.md

### PR DESCRIPTION
**Pull request by antonfirsov**
### Prerequisites
- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
Made several changes on `README.md` to make it up-to date, more compact and easier to follow:
- Removed the alpha disclaimer
- Badges: Removed: stars, forks, issues. Moved the rest to the top of the document so they do not distract the reader
- Added the *Questions?* point
- Moved the **Manual Build** point after the API description
- Moved the "self promotion" part of the API descrtiption to the start of the document
- Removed `Span<T>` stuff from the API description
- Added reference to our samples repository and to our beta blog post

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,14 +85,29 @@ using (Image<Rgba32> image = new Image<Rgba32>(400, 400))
...
 
-For optimized synchronous access within a loop it is recommended that the following methods are used.
+`Rgba32` is our default PixelFormat, equivalent to `System.Drawing Color`. For advanced pixel format usage there are multiple [PixelFormat implementations](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/PixelFormats) available allowing developers to implement their own color models in the same manner as Microsoft XNA Game Studio and MonoGame. 
 
-1. `image.GetRowSpan(y)`
-2. `image.GetRowSpan(x, y)`
+All in all this should allow image processing to be much more accessible to developers which has always been my goal from the start.
 
-For advanced pixel format usage there are multiple [PixelFormat implementations](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/PixelFormats) available allowing developers to implement their own color models in the same manner as Microsoft XNA Game Studio and MonoGame. 
+Check out [this blog post](https://sixlabors.com/blog/announcing-imagesharp-beta-1/) or our [samples repository](https://github.com/SixLabors/ImageSharp/tree/master/samples) for more examples!
```
Comment (tocsoft) [inline README.md]: I've created a new repo for our samples you will want to use this instead.
https://github.com/SixLabors/Samples/tree/master/ImageSharp

**Comment by tocsoft** [comment]
we could do with deleting the samples out of this repo.

File: README.md (hunk 1 above)
Comment (antonfirsov) [reply README.md]: Nice! It wasn't yet here when I was editing the document 😄

Comment (tocsoft) [reply README.md]: yeah I know.. that's why I quickly pushed up a version ;)

**Pull request merged**